ImageSegmentation/app/src/main/java/org/pytorch/imagesegmentation/MainActivity.java

```diff
@@ -0,0 +1,166 @@
+package org.pytorch.imagesegmentation;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.ProgressBar;
+
+import org.pytorch.IValue;
+import org.pytorch.Module;
+import org.pytorch.Tensor;
+import org.pytorch.torchvision.TensorImageUtils;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Map;
+
+public class MainActivity extends AppCompatActivity implements Runnable {
+    private ImageView imageView;
+    private Button buttonSegment;
+    private ProgressBar progressBar;
+    private Bitmap bitmap = null;
+    private Module module = null;
+    private String imagename = "deeplab.jpg";
```

Comment:
style nit: on your choice, for previous demo apps member fields were prefixed with m,
`mImageView`, `mButtonSegment` etc.